.storybook/preview.js

```diff
@@ -1,4 +1,4 @@
-import '@axa-ch/patterns-library-polyfill';
+import './polyfills-ie11.js';
```

Comment:
@raphaellueckl why this?

Reply:
i mean, why not keep our ones?

Reply:
We keep our ones, but "array.split" or something was not supported. But feel free to reduce them. That part caused a lot of work, but in the end, I never fully optimized it, now that you mention it. Good point.

BUT "our ones" are still there, just within that file. :)

Reply:
It is still everything working with just our polyfills. Changed that and clicked trough all our components.

Reply:
I have splitted that polyfill files into 2 imports:
- the additional polyfills for storybook 5.3
- our good old PL polyfills for our components